Dockerfile

```diff
@@ -25,5 +25,13 @@ RUN apk --no-cache add postgresql-libs ca-certificates libxslt jpeg zlib file li
 WORKDIR /data/app
 ADD . .
 
+RUN adduser -S open_event_user
+# Gives permission to open_event_user
+RUN chown -R open_event_user /data/app
+RUN chown -R open_event_user ./static
+RUN chown -R open_event_user ./generated
```

Comment:
This is run at creation of the docker image, so it ran on your machine only. It'll never run on server

Reply:
Then to run on the server, what should be done? Yes while building the image this was done. But in tomorrow PR how does it work?

Reply:
Because that folder is inside the container and these are on host

Reply:
@iamareebjamal - Okay but can you show me the direction about how to solve this now. We want this to run without building the image.?

Reply:
Means what exactly the issue is here. We wont build the image again on server and I have to add these host files permission which are outside the container.

Reply:
Searching for better solution